internal/broadcaster/multi_rate_broadcaster_test.go

```diff
@@ -73,3 +79,92 @@ func TestMultiRateBroadcasterStart(t *testing.T) {
 	}
 
 }
+
+
+func TestMultiKeyRateBroadcaster_Start(t *testing.T) {
```

Comment:
This test is now a duplicate of the test above

Reply:
addressed